src/tests/ftest/interoperability/updown_grade.yaml

```diff
@@ -1,35 +1,34 @@
 hosts:
-  test_servers: 9
-  test_clients: 1
-timeout: 850
+  test_servers:
+    - server-A
```

Comment:
Please note: using the old-format for test_servers, for the base RPMs to run the upgrade.
Will update with new format after the base RPMs using the new format.